.travis.yml

```diff
@@ -1,32 +0,0 @@
-language: node_js
-jobs:
-  include:
-  - node: 17
-    env: NODE_OPTIONS=--openssl-legacy-provider
-install:
-  - npm install
-git:
-  depth: 5
-script:
-  - npm run build
-  - npm run test:ci
-  - npm run lint
```

Comment:
Tiny nitpick: should we add this call to the linter just to be consistent?

Reply:
That was a oversight on my part. Added!